apps/freeablo/faworld/playerfactory.cpp

```diff
@@ -24,61 +25,51 @@ namespace FAWorld
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::baseRingQlvl5), MakeEquipTarget<EquipTargetType::inventory>(1, 2));
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::baseRingQlvl5), MakeEquipTarget<EquipTargetType::inventory>(2, 2));
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::baseAmuletQlvl8), MakeEquipTarget<EquipTargetType::inventory>(1, 3));
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::baseHelm), MakeEquipTarget<EquipTargetType::inventory>(5, 0));
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::baseRags), MakeEquipTarget<EquipTargetType::inventory>(7, 0));
     }
 
     void PlayerFactory::createWarrior(Player* player) const
     {
-        ItemManager& itemManager = ItemManager::get();
-
-        FAWorld::Item item = itemManager.getItemByName("Short Sword");
-        player->mInventory.putItemUnsafe(item, FAWorld::MakeEquipTarget<EquipTargetType::leftHand>());
-
-        item = itemManager.getItemByName("Buckler");
-        player->mInventory.putItemUnsafe(item, FAWorld::MakeEquipTarget<EquipTargetType::rightHand>());
-
-        item = itemManager.getItemByName("Club");
-        player->mInventory.putItemUnsafe(item, FAWorld::MakeEquipTarget<EquipTargetType::inventory>(0, 0));
-
-        item = itemManager.getItemByName("Gold");
-        item.mCount = 100;
-        player->mInventory.putItemUnsafe(item, FAWorld::MakeEquipTarget<EquipTargetType::inventory>(0, 3));
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::shortSword), MakeEquipTarget<EquipTargetType::leftHand>());
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::buckler, BaseItemGenOptions()), MakeEquipTarget<EquipTargetType::rightHand>());
+        player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::club), MakeEquipTarget<EquipTargetType::inventory>(0, 0));
+        {
+            auto item = mItemFactory.generateBaseItem(ItemId::gold);
+            item.mCount = 100;
+            player->mInventory.putItemUnsafe(std::move(item), MakeEquipTarget<EquipTargetType::inventory>(0, 3));
+        }
 
         for (auto i = 0; i < 2; ++i)
         {
-            item = itemManager.getItemByName("Potion of Healing");
-            player->mInventory.putItemUnsafe(item, FAWorld::MakeEquipTarget<EquipTargetType::belt>(i));
+            player->mInventory.putItemUnsafe(mItemFactory.generateBaseItem(ItemId::potionOfHealing), MakeEquipTarget<EquipTargetType::belt>(i));
         }
 
         player->setSpriteClass("warrior");
         player->mAnimation.setAnimation(AnimState::idle, FARender::Renderer::get()->loadImage("plrgfx/warrior/wld/wldst.cl2"));
         player->mAnimation.setAnimation(AnimState::walk, FARender::Renderer::get()->loadImage("plrgfx/warrior/wld/wldwl.cl2"));
+        // loadTestingKit (player);
```

Comment:
?

Reply:
Well that's stuff I use for testing items, but maybe it's better to remove than comment it of course.